src/backend/base/langflow/components/embeddings/text_embedder.py

```diff
@@ -27,26 +36,50 @@ class TextEmbedderComponent(Component):
             info="The message to generate embeddings for.",
         ),
     ]
-
     outputs = [
         Output(display_name="Embedding Data", name="embeddings", method="generate_embeddings"),
     ]
 
     def generate_embeddings(self) -> Data:
-        embedding_model: Embeddings = self.embedding_model
-        message: Message = self.message
+        try:
+            embedding_model: Embeddings = self.embedding_model
+            message: Message = self.message
+
+            # Validate embedding model
+            if not embedding_model:
+                raise ValueError(ERRORS["NO_MODEL"])
```

Comment:
Can we use classes instead ? Eg NoModelError.
NB: if this is to escape the EM ruff rules, this one can be auto-fixed with `ruff --fix --unsafe-fixes`

Reply:
sure!

Reply:
@cbornet first was hardcoded the messages.. I tried that command and didn't work :(
that's why I changed for this constants

Reply:
@cbornet when I'm using classes the custom component is not accepting that..

Reply:
Weird. It always works for me.
Maybe try `uv run ruff --fix --unsafe-fixes` ?
The auto-fix creates an msg variable so you have
```python
msg = "something bad happened"
raise ValueError(msg)
```
This fixes rule EM and at the same time TRY003 is not detected anymore.
But the real fix for TRY003 would be to use a specific error.
Now I find TRY003 a bit harsh so I'm fine with the auto-fix.

Reply:
that worked! Thanks!